app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/login/AuthTextFieldsEnum.kt

```diff
@@ -2,13 +2,19 @@ package com.example.composetemplate.presentation.screens.entry_screens.login
 
 import androidx.compose.foundation.text.KeyboardOptions
 import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.text.input.OffsetMapping
 import androidx.compose.ui.text.input.TransformedText
 import androidx.compose.ui.text.input.VisualTransformation
 import com.example.composetemplate.utils.Constants
 
-enum class LoginScreenEnum {
+enum class LoginScreenStateManagement {
```

Comment:
Rename it to something like AuthScreenState. Login is single item in the enum so it can't be the whole enum name.